css/css-writing-modes/wm-propagation-body-044-ref.html

```diff
@@ -13,6 +13,11 @@
       padding-right: 16px;
       vertical-align: top;
     }
+  div
+    {
+      style=margin-left: -8px;
```

Comment:
style=margin-left?

Reply:
doing things quicky before rushing to dinner results in poor copy-pasta. Sorry.